src/rules/core/id-references-for-certain-feeds-rule-spec.js

```diff
@@ -0,0 +1,157 @@
+const Model = require('../../classes/model');
+const ModelNode = require('../../classes/model-node');
+const OptionsHelper = require('../../helpers/options');
+const IdReferencesForCertainFeedsRule = require('./id-references-for-certain-feeds-rule');
+const ValidationErrorType = require('../../errors/validation-error-type');
+const ValidationErrorSeverity = require('../../errors/validation-error-severity');
+
+describe('IdReferencesForCertainFeedsRule', () => {
```

Comment:
For these rules, see the refactoring of [`#407`](https://github.com/openactive/data-model-validator/pull/407/files#diff-363b70ba4b79c3b30a8ee96c6d795cddde7a92c2a89adc4e37f3664475b28e9c).

Recommend refactoring to separate the request/response concerns (which should be covered by the validation mode)